canonical-web-and-design/ubuntu.com
# Image builder move url

## Done
This is the redo of this work: https://github.com/canonical-web-and-design/ubuntu.com/pull/6985

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/build
- Run a build and check it all works
